test/util/IdTableHelpers.cpp

```diff
@@ -153,32 +155,37 @@ IdTable createRandomlyFilledIdTable(
 }
 
 // ____________________________________________________________________________
-IdTable createRandomlyFilledIdTable(const size_t numberRows,
-                                    const size_t numberColumns,
-                                    const std::vector<size_t>& joinColumns,
-                                    const std::function<ValueId()>& generator) {
+IdTable createRandomlyFilledIdTable(
+    const size_t numberRows, const size_t numberColumns,
+    const std::vector<size_t>& joinColumns,
+    const std::function<ValueId()>& generator,
+    const unsigned int randomNumberGeneratorSeed) {
```

Comment:
These would benefit from the strong `Seed` type.

Reply:
Done, see [abbbe85](https://github.com/ad-freiburg/qlever/pull/1126/commits/abbbe85a8dd2130b9832263b6d18f9ae46d3e6a2).